src/models/client_config.ts

```diff
@@ -10,4 +10,9 @@ export class ClientConfig {
   public api_key?: string;
   public api_url?: string;
   public ui_url?: string;
+}
+
+export function IsValidConfig(toCheck: object): boolean {
+  const config = toCheck as ClientConfig;
+  return config.api_url !== undefined && config.ui_url !== undefined;
```

Comment:
Should this also be checking to make sure the username and api key are defined?

Reply:
My approach was to spot check key fields, since it would be annoying to check everything as we add more config options. Also the credentials will be going away soon